Start QR code scanner from app

di72nn opened the pull request:
In a feat of procrastination I wrote some code for the app to invoke QR-code scanner.
I added a button on one of the wizard pages. Suggestions are welcome.

tcitworld commented on app/src/main/java/fr/gaulupeau/apps/Poche/ui/preferences/ConnectionWizardActivity.java:
```diff
@@ -132,6 +169,25 @@ private ConnectionData parseLoginData(String connectionUri) {
 
+    public void scanQrCode() {
+        try {
+            Intent intent = new Intent("com.google.zxing.client.android.SCAN");
```
Does that enforce a specific QR scanner client ?

tcitworld commented:
Needs rebase to have only the second commit.

di72nn commented:
It was a PITA to set up the app multiple times during development, so I updated this PR. I also added support for password. With these changes one can set up his account in one swift press-the-button-find-the-qr-code-point-the-phone-at-the-code motion.

The intent works with **multiple QR code scanning applications**: a list of apps is presented by Android. I can see "Barcode Scanner" and "Binary Eye" (I don't think I have any other installed).

di72nn commented:
Added `intent.removeExtra(EXTRA_FILL_OUT_FROM_SETTINGS);` which I missed yesterday. Without it the feature didn't work if the wizard wasn't started from settings.
